automod_legacy/swearwords.go

```diff
@@ -60,6 +60,8 @@ var BuiltinSwearWords = map[string]bool{
 	"smegma":      true,
 	"spunk":       true,
 	"tit":         true,
+	"tranny":       true,
+	"trannies":     true,
```

Comment:
Please indent the `true` boolean with the rest of them in the line, seems strange to have them one space out.